platform/brains2/src/duo/main.cpp

```diff
@@ -347,7 +346,6 @@ void process_key(const char k, const char state) {
 }
 
 inline void keys_scan() {
-  AudioNoInterrupts();
```

Comment:
I noticed some flickering issues again, and removing this improves it for me. It also seems to me that disabling interrupts here is not actually necessary?

Reply:
no, every time we change parameters in the synth the interrupts should be disabled. If we want to remove this we have to make sure no synth parameters are changed in any of the key handling functions.

Reply:
But, it's only if it's crucial that all settings take effect simultaneously, right?
https://www.pjrc.com/teensy/td_libs_AudioProcessorUsage.html

In this case, I think it's not that important, since we're fading in and out, and some samples misalignment shouldn't be audible.